### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by ghjer** [Protoqol/Prequel#79]
composer install fails on latest laravel version 6.0.1. Are you guys planning on releasing a version supporting laravel > 6?

```
  Problem 1
    - Installation request for protoqol/prequel ^1.22 -> satisfiable by protoqol/prequel[v1.22].
    - Conclusion: remove laravel/framework v6.0.1
    - Conclusion: don't install laravel/framework v6.0.1
    - protoqol/prequel v1.22 requires laravel/framework ^5.6 -> satisfiable by laravel/framework[5.6.x-dev, 5.7.x-dev, 5.8.x-dev].
    - Can only install one of: laravel/framework[5.6.x-dev, v6.0.1].
    - Can only install one of: laravel/framework[5.7.x-dev, v6.0.1].
    - Can only install one of: laravel/framework[5.8.x-dev, v6.0.1].
    - Installation request for laravel/framework (locked at v6.0.1, required as ^6.0) -> satisfiable by laravel/framework[v6.0.1].
```

**Comment by ZeroGodForce** [comment]
I've literally just had a similar result with 5.8.35

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Can only install one of: composer/composer[dev-master, 1.9.0].
    - Can only install one of: composer/composer[dev-master, 1.9.0].
    - Can only install one of: composer/composer[dev-master, 1.9.0].
    - protoqol/prequel v1.22 requires composer/composer dev-master -> satisfiable by composer/composer[dev-master].
    - Installation request for protoqol/prequel ^1.22 -> satisfiable by protoqol/prequel[v1.22].
    - Installation request for composer/composer (locked at 1.9.0) -> satisfiable by composer/composer[1.9.0].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by QuintenJustus** [comment]
Hey! See #78

**Comment by QuintenJustus** [comment]
Support is confirmed, you can now use version 1.22.1 with Laravel >6.0

**Comment by ZeroGodForce** [comment]
I wasn't able to get it to install via `composer install` on v5.8.35.  I don't know if there was something to do with my local composer references, and it didn't make a difference after I ran `composer clearcache`, so I added it manually into my composer.json as `"protoqol/prequel": "^1.22.1"` and then ran `composer update` which seemed to work.  

As a matter of course, I added it into my `require-dev`, as it's not yet production ready - but I have to say, great work so far!  Looks like this'll turn out to be a really great project :)
